doctum.conf.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+// Prevent e.g. 'Notice: Constant MATH_BIGINTEGER_MONTGOMERY already defined'.
+class MyArrayStore extends Doctum\Store\ArrayStore
+{
+    public function removeClass(Doctum\Project $project, $name)
+    {
+        unset($this->classes[$name]);
+    }
+}
+
+$root = realpath(__DIR__) . '/phpseclib/';
+
+$iterator = Symfony\Component\Finder\Finder::create()
+    ->files()
+    ->name('*.php')
+    ->in($root . 'phpseclib');
+
+$versions = Doctum\Version\GitVersionCollection::create($root)
```

Comment:
We basically want all branches and we do not want to list them explicitly. It is the simplest solution. There was some support in sami for using all branches. However, sami had some issues in case a source file is exactly the same in two branches, e.g. 2.0 and 3.0. Would have to verify this is no longer an issue.